### ISSUE TITLE
Shipping is not showing in checkout magento2?

### ISSUE THREAD

**Issue by ramsampathi** [magento/magento2#2332]
Hi,
Installed the Magento2 ver. 1.0.0-beta in my host.Added some Products for testing purpose.In front end checkout page shipping information is not showing.can u plz give me suggestions for solve this issue...?
![screenshot_mage2](https://cloud.githubusercontent.com/assets/7812791/11113330/f9353d50-8941-11e5-9b2b-b08723482f32.png)

Thanks,
Ramakrishna Sampathi

**Comment by asemenenko** [comment]
Hi @ramsampathi Product in the cart is virtual thus can not be shipped.

**Comment by daim2k5** [comment]
Hi @ramsampathi did the answer helped?

**Comment by QuangAn** [comment]
i have added configurable product but not have shipping step in onepage checkout 
So how to show shipping step ?
http://prntscr.com/b2mw71

**Comment by sakhil9495** [comment]
Hi @ramsampathi ,

I am also faced the same issue. How can i fix the issue. Please help me in fixing the issue

**Comment by pboisvert** [comment]
Does your product have a weight set? If not it will be treated as virtual/downloadable and no shipping will be shown.

**Comment by rfapinkesh** [comment]
![Uploading Screenshot from 2017-01-28 12:41:05.png…]()
I am also faced the same issue. My all products are simple products and also have a weight set.
First time country is missing and after refresh the page, it shows only zip code..
Please help me in fixing the issue....

**Comment by PardeepGrover** [comment]
Hi ,

I am also facing the same issue,Can anyone help on  this to resolve.

Thanks

**Comment by ghost** [comment]
same issue! have simple and configurable product with weight! everything set correctly and shipping methods (UPS and Flat rate - both of them are active and configured) does not shown in checkout page!

**Comment by serefatka** [comment]
any updates regarding this issue? the same problem on magenta v.2.2

**Comment by Prasad2706** [comment]
same issue faced

**Comment by killday** [comment]
Any help on this? In my case the flat rate suddenly stop showing. Magento 2.2.0

**Comment by jotakar** [comment]
I have the same problem Magento 2.3., All other shipping method are shown OK,

**Comment by killday** [comment]
What worked for me is to show the mini cart in the layout for the checkout page and then hiding it again. For some reason when hide the mini cart it hides the shipping method.

**Comment by jotakar** [comment]
Thak you but this is not a valid fix for me. 
I need flate rate shown as the other shipping method does.

**Comment by Janagiramramakrishnan** [comment]
I am using Magento 2.3.4 and a single website. I have created only one configurable product and tried to add the cart and went to the checkout page. But shipping tabs are not displaying. Later created one simple product and add the products both configurable and simple then shipping tabs displayed. I think when we create configurable products and all the associated products generated and the product type is virtual. This is the reason shipping tabs are not displayed.

**Question:**
If my website having only configurable products and all the associated products created/generated automatically the type is virtual. Then how it shipping tabs displayed?

I think this is the bug. when we created the configurable products and all the associated product types are generated in virtual type.
